MagentoEcsFargate/README.md

```diff
@@ -0,0 +1,37 @@
+# Containerizing our Magento stack
+
+This directory contains only architecture and installation steps. For more details, please check the [blog] (https://medium.com/i-love-my-local-farmer-engineering-blog/xxxx) 
+
+## Prerequisites
+1. **NodeJS (v. 12+)** NodeJS is required for running and installing AWS CDK. You can download it [here](https://nodejs.org/en/download/).
+1. **AWS CDK (v. 2.24.1)**: This solution uses AWS CDK for deployment. If you aren't familiar with CDK, please install its [prerequisites](https://cdkworkshop.com/15-prerequisites.html) and follow the  [Python workshop](https://cdkworkshop.com/30-python.html) first.   
+
+## Architecture
+To understand what this solution will deploy, we will start from the original architecture with virtual machine deployed on prem
+
+![](.README_images/on-prem-architecture.png)
+
+The solution uses Amazon ECS on Fargate to host the Magento App, Amazon RDS for MariaDB and Amazon Opensearch as dependencies.
+
+![](.README_images/on-aws.jpg)
+
+
+### !!! This example is provided as a sample and rely on HTTP, which should never be used in production system . Use [encrypted traffic] (https://docs.aws.amazon.com/elasticloadbalancing/latest/application/create-https-listener.html) instead !!!
```

Comment:
This is only for the listener (b/w ALB and Fargate), right? Does the ALB public endpoint still use HTTPS?

I assume that we don't use HTTPs because we have no certificate? Could we clarify it here?

Another point: From reader's perspective, I would probably want this in CDK code. It would be great if there is a link for CDK too (assuming that there is already certificate imported in ACM).